source/Ocl/Converters/OclAttributeAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Octopus.Ocl.Converters
+{
+    [AttributeUsage(AttributeTargets.Property,  AllowMultiple = false)]
+    public sealed class OclAttributeAttribute : Attribute
```

Comment:
Given `Ocl Attribute` already means something specific in this domain, would `OclPropertyAttribute` be more approriate here?

My thinking is that this could be used in future to name ocl attributes _or_ ocl blocks depending on property type and converters used.

Reply:
> Given `Ocl Attribute` already means something specific in this domain
I considered this, but I went with it being specific to OCL attributes because
a) That's currently all it's implemented for
b) Any future additions may will be specific to only attributes.  

I took inspiration from the JSON serialization attributes here.  

We can always create an `OclBlockAttribute` as well and have the two inherit from a common parent.

Reply:
Maybe it would be easer to jump on a zoom when we're both free. Adding this system attribute to a prop has no direct control over whether the prop gets converted to an Ocl Attribute or an Ocl Block. That depends entirely on which converter picks it up.